pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -53,7 +54,7 @@ buildPythonPackage rec {
     gunicorn
     pytest-mock
     pytest-xdist
-    pytestCheckHook
+    (pytestCheckHook.override { enforcedPythonPackages = [ pytest_6_1 ]; })
```

Comment:
maybe introduce a `pytestCheckHook_6_1`? We used to have that. Note it is a bit tricky as where to define it; cross should still work.